src/QsCompiler/Compiler/Templates/CppInterop.cs

```diff
@@ -159,17 +159,17 @@ public string CppCliValueType()
 
         public string CliValueVariableName()
         {
-            return this.Name + "CliValue";
+            return "v" + this.Name + "CliValue";
```

Comment:
What's the motivation to add a "v" at the beginning?

Reply:
the compiling seems to fail if variables start with uppercase. I think from a lint check or something?

Reply:
Can we just disable linting? Seems wrong that it should be run on autogenerated code. And if not, the intent might be more clear by using `this.Name.ToLower()` and commenting the purpose behind it.

Reply:
Lint is only enabled when using a driver in a similar setup that the [reference driver](https://github.com/microsoft/qsharp-runtime/blob/main/src/Qir/Samples/StandaloneInputReference/qir-driver.cpp) uses. Having said that, C++ source code generated by this component will not be subject to linting so I think the "v" prefix should be removed.

For testing purposes, the [.clang-tidy](https://github.com/microsoft/qsharp-runtime/blob/main/src/Qir/Samples/StandaloneInputReference/.clang-tidy) file can be edited to have nothing and that should effectively disable linting.

Reply:
ok I will remove, the reason I didn't do this.Name.ToLower() is just because in theory it is possible to have name clashes if you do that